spec/spec_helper.rb

```diff
@@ -15,7 +15,7 @@
   Capybara.default_driver = :selenium
 
   config.before(:suite) do
-    $vite_pid = spawn("yarn dev --port 5678")
+    $vite_pid = spawn("yarn dev --port 5678 --force")
```

Comment:
`--force` tells `yarn dev` to ignore its cached npm packages.  Without `--force`, we might get false positives and false negatives when running the tests locally, since it might be looking at a different version of an npm package than what is in yarn.lock.